migrations/2023-05-12-103127_payment_intent.sql/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE payment_intent ADD COLUMN meta_data jsonb;
```

Comment:
There already is a column called metadata for this table, can you use the same column and not create another instead.

Reply:
@Narayanbhat166 this new column is added for backward_compatibility. once things are stable, we will remove old one and update the existing one